### PR TITLE
Fix godoc for client code

### PR DESCRIPTION
**Pull request by Xe**
ex:

```go
/* FindPetsByStatus: Finds Pets by status

Multiple status values can be provided with comma separated strings
*/
func (a *Client) FindPetsByStatus(params *FindPetsByStatusParams, authInfo client.AuthInfoWriter) (*FindPetsByStatusOK, error) {
...
```

### CODE REVIEW

File: generator/templates/client/client.gotmpl [hunk 1]
```diff
@@ -32,7 +32,7 @@ type Client struct {
   formats strfmt.Registry
 }
 
-{{ range .Operations }}/*{{ if .Summary }}{{ .Summary }}{{ if .Description }}
+{{ range .Operations }}/* {{ if .Summary }}{{ pascalize .Name }}: {{ .Summary }}{{ if .Description }}
```
Comment (casualjim) [inline generator/templates/client/client.gotmpl]: I think it really wants to not have that space between /* and {{ 
it's probably golint or go vet that would be unhappy about the space

Comment (Xe) [inline generator/templates/client/client.gotmpl]: This is `/*FindPetsByStatus: Finds Pets by status` vs `/* FindPetsByStatus: Finds Pets by status`.

**Comment by Xe** [comment]
```console
$ swagger generate client swagger.json && golint ./...
2016/01/16 22:05:58 building a plan for generation
2016/01/16 22:05:58 planning definitions
2016/01/16 22:05:58 planning operations
2016/01/16 22:05:59 grouping operations into packages
2016/01/16 22:05:59 planning meta data and facades
2016/01/16 22:05:59 rendered model template: Pet
2016/01/16 22:05:59 rendered model template: Order
2016/01/16 22:05:59 rendered model template: Category
2016/01/16 22:05:59 rendered model template: User
2016/01/16 22:05:59 rendered model template: Tag
2016/01/16 22:05:59 rendered model template: ApiResponse
2016/01/16 22:05:59 rendered client parameters template: pet.AddPetParameters
2016/01/16 22:05:59 rendered client responses template: pet.AddPetResponses
2016/01/16 22:05:59 rendered client parameters template: pet.DeletePetParameters
2016/01/16 22:05:59 rendered client responses template: pet.DeletePetResponses
2016/01/16 22:05:59 rendered client parameters template: pet.FindPetsByStatusParameters
2016/01/16 22:05:59 rendered client responses template: pet.FindPetsByStatusResponses
2016/01/16 22:05:59 rendered client parameters template: pet.FindPetsByTagsParameters
2016/01/16 22:05:59 rendered client responses template: pet.FindPetsByTagsResponses
2016/01/16 22:05:59 rendered client parameters template: pet.GetPetByIDParameters
2016/01/16 22:05:59 rendered client responses template: pet.GetPetByIDResponses
2016/01/16 22:05:59 rendered client parameters template: pet.UpdatePetParameters
2016/01/16 22:05:59 rendered client responses template: pet.UpdatePetResponses
2016/01/16 22:05:59 rendered client parameters template: pet.UpdatePetWithFormParameters
2016/01/16 22:05:59 rendered client responses template: pet.UpdatePetWithFormResponses
2016/01/16 22:05:59 rendered client parameters template: pet.UploadFileParameters
2016/01/16 22:05:59 rendered client responses template: pet.UploadFileResponses
2016/01/16 22:05:59 rendered operation group client template: pet.PetClient
2016/01/16 22:05:59 rendered client parameters template: store.DeleteOrderParameters
2016/01/16 22:05:59 rendered client responses template: store.DeleteOrderResponses
2016/01/16 22:05:59 rendered client parameters template: store.GetInventoryParameters
2016/01/16 22:05:59 rendered client responses template: store.GetInventoryResponses
2016/01/16 22:05:59 rendered client parameters template: store.GetOrderByIDParameters
2016/01/16 22:05:59 rendered client responses template: store.GetOrderByIDResponses
2016/01/16 22:05:59 rendered client parameters template: store.PlaceOrderParameters
2016/01/16 22:05:59 rendered client responses template: store.PlaceOrderResponses
2016/01/16 22:05:59 rendered operation group client template: store.StoreClient
2016/01/16 22:05:59 rendered client parameters template: user.CreateUserParameters
2016/01/16 22:05:59 rendered client responses template: user.CreateUserResponses
2016/01/16 22:05:59 rendered client parameters template: user.CreateUsersWithArrayInputParameters
2016/01/16 22:05:59 rendered client responses template: user.CreateUsersWithArrayInputResponses
2016/01/16 22:05:59 rendered client parameters template: user.CreateUsersWithListInputParameters
2016/01/16 22:05:59 rendered client responses template: user.CreateUsersWithListInputResponses
2016/01/16 22:05:59 rendered client parameters template: user.DeleteUserParameters
2016/01/16 22:05:59 rendered client responses template: user.DeleteUserResponses
2016/01/16 22:05:59 rendered client parameters template: user.GetUserByNameParameters
2016/01/16 22:05:59 rendered client responses template: user.GetUserByNameResponses
2016/01/16 22:05:59 rendered client parameters template: user.LoginUserParameters
2016/01/16 22:06:00 rendered client responses template: user.LoginUserResponses
2016/01/16 22:06:00 rendered client parameters template: user.LogoutUserParameters
2016/01/16 22:06:00 rendered client responses template: user.LogoutUserResponses
2016/01/16 22:06:00 rendered client parameters template: user.UpdateUserParameters
2016/01/16 22:06:00 rendered client responses template: user.UpdateUserResponses
2016/01/16 22:06:00 rendered operation group client template: user.UserClient
2016/01/16 22:06:00 rendered client facade template: client.SwaggerPetstoreClient
```

**Comment by casualjim** [comment]
lgtm

File: generator/templates/client/parameter.gotmpl [hunk 2]
```diff
@@ -45,7 +45,7 @@ type {{ pascalize .Name }}Params struct {
 
 {{ range .Params }}
 // With{{ pascalize .Name }} adds the {{ camelize .Name  }} to the {{ humanize $.Name }} params
-func ({{ $.ReceiverName }} *{{ pascalize $.Name }}Params) With{{ pascalize .Name }}({{ camelize .Name  }} {{ if and (not .IsArray) (not .HasDiscriminator) (or .IsNullable  ) }}*{{ end }}{{ if not .IsFileParam }}{{ .GoType }}{{ else }}os.File{{ end }}) *{{ pascalize $.Name }}Params {
+func ({{ $.ReceiverName }} *{{ pascalize $.Name }}Params) With{{ pascalize .Name }}({{ pascalize .Name  }} {{ if and (not .IsArray) (not .HasDiscriminator) (or .IsNullable  ) }}*{{ end }}{{ if not .IsFileParam }}{{ .GoType }}{{ else }}os.File{{ end }}) *{{ pascalize $.Name }}Params {
```
Comment (casualjim) [inline generator/templates/client/parameter.gotmpl]: please undo this. lowercased param names!

**Comment by Xe** [comment]
```console
$ golint ./...
client/pet/delete_pet_parameters.go:41:37: method parameter petId should be petID
client/pet/get_pet_by_id_parameters.go:33:38: method parameter petId should be petID
client/pet/update_pet_with_form_parameters.go:49:45: method parameter petId should be petID
client/pet/upload_file_parameters.go:57:38: method parameter petId should be petID
client/store/delete_order_parameters.go:33:41: method parameter orderId should be orderID
client/store/get_order_by_id_parameters.go:33:42: method parameter orderId should be orderID
```

**Comment by casualjim** [comment]
LGTM perhaps

**Pull request merged**